plugin/postgres/plugin.go

```diff
@@ -21,7 +21,8 @@
 //        "pg_user":"username-for-postgres",
 //        "pg_password":"password-for-above-user",
 //        "pg_host":"hostname-or-ip-of-pg-server",
-//        "pg_port":"port-above-pg-server-listens-on"
+//        "pg_port":"port-above-pg-server-listens-on",
+//        "pg_database": "name-of-db-to-backup"
```

Comment:
We should add a note that `pg_database` is optional, and explain what the semantics are when present and when omitted.